### PR TITLE
add initial version of README file

### PR DESCRIPTION
**Pull request by arnisoph**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        flatten: true
+        post-deploy:
+          - name: reload prometheus
+            cmd: ["curl", "-s", "-X", "POST", "http://localhost:9090/-/reload"]
+  prometheus.yml: |-
+    global:
+      #scrape_interval: 1m
+      #evaluation_interval: 1m
+      external_labels:
+    ...
```
Comment (hikhvar) [inline README.md]: This makes the example not working. How about providing a complete example in a distinct file. That could be  applied via `kubectl apply -f http://github.com/inovex/trovilio/....`. That should make things more clear.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
+# trovilo
+
+Trovilo collects and prepares files from Kubernetes ConfigMaps for Prometheus & friends
```
Comment (hikhvar) [inline README.md]: I think here is a `.` missing.

**Review by hikhvar — CHANGES_REQUESTED** [review]
Nice! I think we should provide a working example.

File: README.md (hunk 1 above)
Comment (arnisoph) [reply README.md]: done, please check! :)

**Comment by arnisoph** [comment]
I've updated the PR. :)

**Review by hikhvar — APPROVED** [review]

**Pull request merged**